composer.json

```diff
@@ -18,16 +18,13 @@
     },
     "config": {
         "sort-packages": true,
-        "platform": {
-            "php": "7.4.99"
-        },
```

Comment:
This should stay, but needs updating.

Reply:
@Ocramius should be 8.0.99 ?

Reply:
Yep, and remember to do at least a `composer update nothing` for the lockfile 👍

Reply:
done